frontend/public/components/storage-class.tsx

```diff
@@ -1,23 +1,25 @@
 import * as React from 'react';
+import * as _ from 'lodash-es';
 
 import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
 import { Cog, detailsPage, navFactory, ResourceCog, SectionHeading, ResourceLink, ResourceSummary } from './utils';
-import { fromNow } from './utils/datetime';
 // eslint-disable-next-line no-unused-vars
-import { K8sResourceKindReference } from '../module/k8s';
+import { K8sResourceKind, K8sResourceKindReference } from '../module/k8s';
 
 export const StorageClassReference: K8sResourceKindReference = 'StorageClass';
 
 const menuActions = [Cog.factory.ModifyLabels, Cog.factory.ModifyAnnotations, Cog.factory.Edit, Cog.factory.Delete];
 
+const isDefaultClassAnnotation = 'storageclass.beta.kubernetes.io/is-default-class';
+const isDefaultClass = (storageClass: K8sResourceKind) => _.get(storageClass, ['metadata', 'annotations', isDefaultClassAnnotation], 'false');
```

Comment:
Based on the [k8s docs](https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/):
>The default StorageClass has an annotation storageclass.kubernetes.io/is-default-class set to true. Any other value or absence of the annotation is interpreted as false

In that case we need to check for the annotation to equal `true`:
```js
_.isEqual(_.get(storageClass, ['metadata', 'annotations', isDefaultClassAnnotation], false), true)
```

Reply:
I'd rather have the client be "dumb" and just show the annotation value as-is. Then if a new value is ever allowed for the annotation, it just shows up in the UI. Also if we show the value as "True" in the UI, it might make someone worry that it's incorrectly set since it must be "true" as lowercase in the annotation.

In general, I want to get away from interpreting values. In this case, it's a simple check, but over time these things can break as features are added to kube. API compatibility doesn't prevent new values from being added or validation relaxed, which has caused bugs for us in the past.

Reply:
OK I see your point, just wanted to make sure, cause is user wont be aware of the fact that the value has to be set to `true` he could be surprised that the annotation has no effect